src/components/Navbar.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { Link } from 'gatsby';
+import Header from './Header';
+
+import {
+  navBarStyle,
+  navBarLinks,
+  navBarLinksContainer
+} from '../styles/navBar.module.scss';
+
+const Navbar = () => (
+  <nav className={navBarStyle}>
+      <Header/>
+      <div className={navBarLinksContainer}>
+        <Link className={navBarLinks} to='/articles'>Articles</Link>
```

Comment:
It would be slightly better for screenreaders if these links would be wrapped in list in addition. So
`<nav><ul><li><Link className..... /></li><li><Link className.../> .... </ul></nav>`

It would also make it easier to ad din sub navigations later if we want to 😁